src/utilities/Utilities.jl

```diff
@@ -0,0 +1,11 @@
+module Utilities
```

Comment:
Why we need a single module for a few line of code? This is too heavy.

Reply:
There will be several other methods in the module of utilities later.

Reply:
Then define it when you have these utilities. Do not do any pre-design, this is bad.

Reply:
let's remove this module, it is not useful at all. You are increasing the complexity of such a simple binding package in very early stage, this is not a good engineering habit IMHO.